packs/wix.js

```diff
@@ -0,0 +1,21 @@
+
+const UIStrings = {
+/** Additional description of a Lighthouse audit that tells the user to optimize image formats, in the context of the Wix CMS platform. */
+  "modern-image-formats": "Upload images using Wix Media Manager to ensure they are automatically served as WebP. Find [more ways to optimize](https://support.wix.com/en/article/site-performance-optimizing-your-media) your site's media.",
+  /** Additional description of a Lighthouse audit that tells the user to defer loading of non-critical third-party libraries, in the context of the Wix CMS platform. */
+  "render-blocking-resources": "When [adding third-party code](https://support.wix.com/en/article/site-performance-using-third-party-code-on-your-site) in the Custom Code tab of your site's dashboard, make sure it's deferred or loaded at the end of the code body. Where possible, use Wix’s [integrations](https://support.wix.com/en/article/about-marketing-integrations) to embed marketing tools on your site. ",
+  /** Additional description of a Lighthouse audit that tells the user how they can improve performance by avoiding animated images and using videos where possible, in the context of the Wix CMS platform. */
+  "efficient-animated-content": "Use videos instead of GIFs to showcase animated content for higher quality and faster load time. Place videos inside VideoBoxes, customize them using Video Masks or add Transparent Videos. [Learn more](https://support.wix.com/en/article/wix-video-about-wix-video).",
```

Comment:
Surrounding product names with backticks
```suggestion
  "efficient-animated-content": "Place videos inside `VideoBoxes`, customize them using `Video Masks` or add `Transparent Videos`. [Learn more](https://support.wix.com/en/article/wix-video-about-wix-video).",
```